### ISSUE TITLE
EasyLink build fails against the CC13x2

### ISSUE THREAD

**Issue by rei-vilo** [energia/cc13x2-core#12]
See https://github.com/energia/EasyLink/issues/17

**Comment by lyusupov** [comment]
@rei-vilo 

    Hello!
Could you, please, advise - what particular **package_index.json** do you use with 'Boards Manager' to install board support for LaunchPad CC1352 ?

**Comment by robertinant** [comment]
Yes, this is correct. A new cc13x2 package will be released soon to support EasyLink.

**Comment by robertinant** [comment]
@lyusupov, there is no public release yet for the cc13x2. You can expect it to show up in the board manager in the next two weeks.

**Comment by lyusupov** [comment]
@robertinant 

Thank you for the answer!

I keep an eye on your progress with port of **Energia** onto CC13X2 family since last summer.
I am waiting patiently for appearance of very first releases or candidates.
I've noticed that you've already deployed both Core ( 5.29.0 ) and GCC toolchain(s) ( 6.3.1-20170620 and 8.3.1-20190703 ) onto Amazon Cloud. - that's why I decided that an appropriate **package_index.json** does already exist,  at least in **Beta** stage...

**Comment by lyusupov** [comment]
> there is no public release yet for the cc13x2. You can expect it to show up in the board manager in the next two weeks.

Are there any news for the topic ?

**Comment by lyusupov** [comment]
@robertinant 
Are there any new estimates for the cc13x2 availability in the board manager ?
